### ISSUE TITLE
Swapping Mining Drone variations throws an exception

### ISSUE THREAD

**Issue by MarHelmer** [pyfa-org/Pyfa#1773]
## Bug Report
Attempting to swap variations of Mining Drones on those in the Drone Bay throws an exception.  This seems to occur no matter whether the drones are checked off as active, with every combination of starting and ending variations, and at any quantity.  This error even occurs when swapping to the same type of drone that is already in the Drone Bay.

### Expected behavior:
Swapping drone types should successfully change the type of drone in the Drone Bay.

### Actual behavior:
Attempting to swap drone types results in an error.

### Detailed steps to reproduce:
1. Add Mining Drones of any type and quantity to the Drone Bay of any ship fitting.
2. Right-click the Mining Drone(s) and attempt to swap to any type of Mining Drone—_including the type already in the Drone Bay_.
3. An error message dialog box will appear.

### Fits involved in EFT format (Edit > To Clipboard > EFT):
All fits seem to have this error.

### Release or development git branch? Please note the release version or commit hash:
2.5.0

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
Windows 10

### Other relevant information:
Here's the error message:

```
pyfa vv2.5.0
EVE Data Version: 1399820 (2018-10-09 18:27:02)

OS version: Windows-10-10.0.17134-SP0
Python version: 3.6.1 (v3.6.1:69c0db5, Mar 21 2017, 17:54:52) [MSC v.1900 32 bit (Intel)]
wxPython version: 4.0.3 (wxWidgets 3.0.5)
SQLAlchemy version: 1.1.10
Logbook version: 1.0.0
Requests version: 2.18.4
Dateutil version: 2.6.0

####################

Traceback (most recent call last):
  File "C:\Users\holme\Documents\Sync\Git\blitzmann\Pyfa\gui\builtinContextMenus\metaSwap.py", line 131, in handleModule
  File "C:\Users\holme\Documents\Sync\Git\blitzmann\Pyfa\gui\fitCommands\guiMetaSwap.py", line 47, in __init__
NotImplementedError

####################

Traceback (most recent call last):
  File "C:\Users\holme\Documents\Sync\Git\blitzmann\Pyfa\gui\builtinContextMenus\metaSwap.py", line 131, in handleModule
  File "C:\Users\holme\Documents\Sync\Git\blitzmann\Pyfa\gui\fitCommands\guiMetaSwap.py", line 47, in __init__
NotImplementedError

```

**Comment by blitzmann** [comment]
I can reproduce the initial error for variations.

> ...Uh, I think the Drone Bay is borked. "Remove Drone" and "Remove Drone Stack" also throw errors.

Can you try again? What errors is this throwing? They seem to be working for me

**Comment by MarHelmer** [comment]
I've since rebooted my PC (wifi was acting up, you know how it is), and now the "Remove Drone" and "Remove Drone Stack" commands work just fine.  Please ignore that! :)

**Comment by blitzmann** [comment]
Just pushed a fix out. Was planning on making a point release tonight, but it's going to have to wait until tomorrow - should include some other fixes that have been made since 2.5.0
